configure.ac

```diff
@@ -44,59 +58,122 @@ AC_PROG_CPP
 
 # Find the Fortran compiler.
 AC_PROG_FC
-AC_PROG_F77
 
 # Find the install program.
 AC_PROG_INSTALL
 
 # Is bats installed? It is required for testing.
-AC_CHECK_PROGS([FMS_BATS], [bats])
-if test -z "$FMS_BATS"; then
-   AC_MSG_ERROR([Cannot find bats utility. Install bats and try again.])
+AC_CHECK_PROGS([BATS], [bats])
+AM_CONDITIONAL([MAKE_CHECK_ENABLED], [test -n "$BATS"])
```

Comment:
Not clear what MAKE_CHECK_ENABLED would be for. Also this would not be the correct way to set it. Also also, we are eliminating the BATS check.

Reply:
This was a way to _not_ allow `make check`, or more specifically have `make check` do a no-op.  As we are removing `bats` from the check scripts, this section will no longer be needed.  (This commit was done before the `bats` removal changes started.